src/components/activities/ActivitiesList.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
```

Comment:
```suggestion
import React from "react";
```

`Component` is not needed